pkg/koordlet/resourceexecutor/updater.go

```diff
@@ -444,3 +521,18 @@ func MergeConditionIfCPUSetIsLooser(oldValue, newValue string) (string, bool, er
 	merged := v.Union(old)
 	return merged.String(), true, nil
 }
+
+func ParseEventForAuditLog(event *audit.Event, verbose int) {
```

Comment:
This function does not only parse the event but also does the logging. We can call it `AuditLogForEvent()`.